sktime/registry/tests/test_lookup.py

```diff
@@ -98,7 +98,13 @@ def test_all_tags(estimator_scitype):
 
     # there should be at least one tag returned
     # exception: scitypes which we know don't have tags associated
-    if estimator_scitype not in SCITYPES_WITHOUT_TAGS:
+    if (
```

Comment:
see below, I would do this differently. Agree that this needs fixing.

Reply:
Changed the if: to your suggested version above.